contracts/MoneyBoxUser.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.20;
+
+import {MoneyBox} from "./MoneyBox.sol";
+import {Create2} from "@openzeppelin/contracts/utils/Create2.sol";
+
+abstract contract MoneyBoxUser {
+
+    function _calculateMoneyBoxAddress(address _creator, bytes32 _salt, address _token) internal pure returns (address) {
+
+        return Create2.computeAddress(
```

Comment:
Why can't we call Create2.computeAddress direclty in the other contract? What is the MoneyBoxUser wrapper for?

Reply:
ok, can do it that way

Reply:
on second thought looking at the changes the fact that you have to hash some stuff together is an implementation detail you can hide, taking it out will make the code a bit noisier. So mildly opposed to doing it, but OK either way.

Reply:
Okay, Should we then make it a library contract, instead of an abstract contract?

Reply:
I may be wrong but I thought a library would be deployed as another contract, which is more gas and more complicated deployment. We don't want that, we just want to add a feature to the contract that calls it.

Reply:
OK, for consistency I'll name/format it like CreateChildren.createChildren, something like `CalculateMoneyBoxAddress._calculateMoneyBoxAddress`

Reply:
Done in [1ebd8bc](https://github.com/RealityETH/subjectivocracy/pull/98/commits/1ebd8bc291cd2efbc98025ad0853e1e5663f50ae)